common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -140,6 +140,14 @@ class CapaFields(object):
         scope=Scope.settings,
         default=default_reset_button
     )
+    require_answer_before_check = Boolean(
+        display_name=_("Require answer before check"),
```

Comment:
Is length a concern? If not, can we make this "Require Answer Before Enabling Check Button"?
And please change to initial caps to match other advanced settings.